README.md

```diff
@@ -1,3 +1,51 @@
+## What do this branch?
```

Comment:
Can you remove the documentation from the readme and instead and some docs on how to use this feature to the /docs folder?